.pre-commit-config.yaml

```diff
@@ -141,3 +141,20 @@ repos:
           '--disable',
           'dangling-hyphen,line-too-long',
         ]
+  - repo: https://github.com/golangci/golangci-lint
+    rev: v1.59.0
+    hooks:
+      # no built-in support for multiple go.mod
+      # https://github.com/golangci/golangci-lint/issues/828
+      - id: golangci-lint
+        name: golangci-lint-arrow
+        entry: bash -c 'cd go/arrow && golangci-lint run --fix --disable-all -E goimports -E gofmt'
```

Comment:
Can we create a common configuration file instead of specifying `--disable*`/`--enable*`?